packages/jaeger-ui/src/components/App/TopNav.tsx

```diff
@@ -108,8 +108,7 @@ const itemsGlobalLeft: MenuProps['items'] = [
 ];
 
 export function TopNavImpl(props: Props) {
-  const { config, router } = props;
-  const { pathname } = router.location;
+  const { config, pathname } = props;
```

Comment:
Does this work if Jaeger is running not at the root but at a prefix, like `/jaeger/search...`?